### DISCUSSION TITLE
Navigation bar partition

### DISCUSSION THREAD

**Discussion by ianzone** [web-infra-dev/rspress#1564] [Ideas]
<img width="494" alt="image" src="https://github.com/user-attachments/assets/9a99049f-f1b6-414c-b4d9-335da4b8cfa4">

当前导航键和功能键都集中在导航栏右边，比较冗长。希望跟docusaurus一样将导航键集中在左边，右边放功能键和社交媒体。

**Comment by Timeless0911** [comment]
If you have special requirements for built-in theme, you can customize yourself.
